manifest.emulator.json

```diff
@@ -0,0 +1,54 @@
+{
+  "description": "Finding Facebook Pixels",
+  "author": "Mozilla Rally",
+  "manifest_version": 3,
+  "name": "Facebook Pixel Hunt",
+  "version": "0.2.6",
+  "homepage_url": "https://github.com/mozilla-rally/facebook-pixel-hunt#readme",
+  "permissions": [
+    "idle",
+    "webNavigation",
+    "webRequest",
+    "storage",
+    "unlimitedStorage",
+    "cookies"
+  ],
+  "host_permissions": [
+    "<all_urls>"
+  ],
+  "background": {
+    "service_worker": "dist/background-loader.js"
+  },
+  "content_scripts": [
+    {
+      "matches": [
+        "<all_urls>"
+      ],
+      "js": [
+        "dist/browser-polyfill.min.js",
+        "dist/webScience/pageManager.content.js",
+        "dist/webScience/pageNavigation.content.js"
+      ]
+    },
+    {
+      "matches": [
+        "https://members.rally.mozilla.org/*",
```

Comment:
Does this work against the live site when firebase sdk is in emulator mode? I think it might not, we should remove this if not.

Reply:
I've now removed the prod domain. It should though work with both domains.